src/main/java/dk/bec/polonez/reservationsystem/service/AuthService.java

```diff
@@ -0,0 +1,33 @@
+package dk.bec.polonez.reservationsystem.service;
+
+import dk.bec.polonez.reservationsystem.model.User;
+import dk.bec.polonez.reservationsystem.repository.UserRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+public class AuthService {
+
+    private final UserRepository userRepository;
+
+    public AuthService(UserRepository userRepository) {
```

Comment:
**advice** You can consider using Lombok lib that can generate constructor for you (@AllArgsConstructor or @RequiredArgsConstructor). Feel free to ignore this comment, it's not mandatory.